pyproject.toml

```diff
@@ -1,5 +1,9 @@
 [build-system]
-requires = ["setuptools>=42", "wheel", "scikit-build"]
+requires = [
+    "setuptools>=42",
+    "wheel",
+    "scikit-build @ git+https://github.com/scikit-build/scikit-build.git@739238cfdc7224e02651f94b41f2aea1fd84b519",
+]
```

Comment:
May also be worth updating https://github.com/scikit-build/ninja-python-distributions/blob/master/docs/make_a_release.rst to reference this commit.

Reply:
~~This is only temporary - once the release is out, we'll update this to `scikit-build >= 0.12`~~ Oh, I see what you mean. Yes, that would be very useful to mention, I have to look this syntax up every single time I use it. Ideally in the future, though, we should probably make a "beta" or "RC" release on pypi, then have a job that can run with dev releases enabled, maybe even automatically (will be looking through the resources you sent).